prototypes/ScheduledMerges.hs

```diff
@@ -167,35 +209,40 @@ mergePolicyForLevel 1 [] = MergePolicyTiering
 mergePolicyForLevel _ [] = MergePolicyLevelling
 mergePolicyForLevel _ _  = MergePolicyTiering
```

Comment:
Good point. This means that inserts on a table with a union is cheaper than on one without a union, since we do more work on levelling merges than tiering merges. So potentially we could even spend the "saved" credits on merging the union? These credits are probably well spent, as they help to bring down the cost of lookups, but this also further muddies the separation between the progress of union and non-union (level) merges. :thinking: